test/e2e_node/node_shutdown_linux_test.go

```diff
@@ -510,5 +510,5 @@ func isPodShutdown(pod *v1.Pod) bool {
 		}
 	}
 
-	return pod.Status.Message == podShutdownMessage && pod.Status.Reason == podShutdownReason && hasContainersNotReadyCondition
+	return pod.Status.Message == podShutdownMessage && pod.Status.Reason == podShutdownReason && hasContainersNotReadyCondition && pod.Status.Phase == v1.PodFailed
```

Comment:
Fixing the above will probably make this flaky, so force the failure condition in the test (create a pod that always exits 1)

Reply:
if this specific test never expects a successful pod, that makes sense to me